### PR TITLE
PO-38: Lights turning flow changed

### PR DESCRIPTION
**Pull request by itolegmysin**

### CODE REVIEW

File: src/locales/en/translation.json [hunk 1]
```diff
@@ -674,7 +674,8 @@
             "users": "Current users:"
         },
         "dashboard": {
-            "alllightsoff": "All lights off",
+            "turnofflights": "Turned {{count}} lights off",
```
Comment (iOlivier) [inline src/locales/en/translation.json]: Should be "**Turn** {{count}} lights off"

File: src/locales/nl/translation.json [hunk 2]
```diff
@@ -611,7 +611,8 @@
             "users": "Huidige gebruikers:"
         },
         "dashboard": {
-            "alllightsoff": "Alle lichten uit",
+            "turnofflights": "Gedraaid {{count}} lichten uit",
```
Comment (iOlivier) [inline src/locales/nl/translation.json]: I smell some Google Translate here 😅
```
"turnofflights": "Zet {{count}} lichten uit",
"alllightsoff": "Alle lichten staan uit",
```

File: src/pages/dashboard/index.html [hunk 3]
```diff
@@ -124,8 +124,12 @@ <h3 class="box-title">${'pages.dashboard.floors' & t}</h3>
                 <div class="floor" repeat.for="floor of floors">
                     <div class="row-header">
                         <h3 class="active-lights">${floor.name} (${floor.activeLights.length})</h3>
```
Comment (iOlivier) [inline src/pages/dashboard/index.html]: You can also remove the number with active lights after the room name.

File: src/locales/nl/translation.json (hunk 2 above)
Comment (itolegmysin) [reply src/locales/nl/translation.json]: Yeah 😅, I thought it would be correct 😄

**Pull request merged**